### PR TITLE
refine bazel-build/examples cpp-tutorial README files

### PR DESCRIPTION
**Pull request by pengwa**

### CODE REVIEW

File: cpp-tutorial/stage3/README.md [hunk 1]
```diff
@@ -12,7 +13,8 @@ cc_library(
 )
 ```
 
-Then to use the ```cc_library``` above we add an extra dependency in the form of //[package-path]:[target-name].
+To use our ```hello-time``` libary, an extra dependency is added in the form of //[package-path]:[target-name], in this case, it's ```//lib:hello-time```
```
Comment (jin) [inline cpp-tutorial/stage3/README.md]: while you are here, do you mind changing //[package-path]:[target-name] to `//path/to/package:target_name`?

Comment (pengwa) [reply cpp-tutorial/stage3/README.md]: yes, sure.

**Comment by pengwa** [comment]
done.

**Comment by jin** [comment]
Thank you so much!

**Pull request merged**